src/test/compile-fail/promoted_div_by_zero.rs

```diff
@@ -1,6 +1,6 @@
 #![allow(const_err)]
 
-// error-pattern: attempt to divide by zero
+// error-pattern: referenced constant has errors
```

Comment:
oh no... this is a regression. This test now fails to compile.

I believe to unregress this you have to change https://github.com/rust-lang/rust/blob/9409c208a9a70eb6194c7502979843f42d6fed1a/src/librustc_mir/transform/const_prop.rs#L440 to use `report_as_lint` instead of `report_as_error` if `let ConstKind::Value(ConstValue::Unevaluated(_, _, Some(_))) = constant.literal.val`

Reply:
The regression is back

Reply:
Ohh, did it wrong in the rebase.